### ISSUE TITLE
Add typespecs `t:t/0` for structs

### ISSUE THREAD

**Issue by sitch** [beam-community/ua_parser#34]
Add typespecs `t:t/0` for structs

**Comment by sitch** [comment]
Currently you get things like:

```shell
:0:unknown_type
Unknown type: UAParser.Device.t/0.
________________________________________________________________________________
:0:unknown_type
Unknown type: UAParser.OperatingSystem.t/0.
________________________________________________________________________________
:0:unknown_type
Unknown type: UAParser.UA.t/0.
```
with dialyzer if you try to use the colloquial typescript type def

**Comment by doomspork** [comment]
This was closed via #36 and #37
